doc/changes/latest.rst

```diff
@@ -43,6 +43,9 @@ Enhancements
 
 Changes
 -------
+- :bdg-dark:`Code` Warn the user :func:`nilearn.interfaces.fmriprep.load_confounds_scrub.extract_outlier_regressors` if ``sample_mask`` is empty (all volumes are scrubbed) (:gh:`4558` by `Victoria Shevchenko`_).
+
+- :bdg-dark:`Code` Throw error in :func:`nilearn.signal._handle_scrubbed_volumes` if ``sample_mask`` is empty (:gh:`4558` by `Victoria Shevchenko`_).
```

Comment:
```suggestion
- :bdg-dark:`Code` Throw error if ``sample_mask`` is empty when scrubbing an fMRI time series (:gh:`4558` by `Victoria Shevchenko`_).
```

I suspect the doc build was failing because we were referencing a private function: not naming the function should solve this (also private function can be renamed at any time so this way we do not have to update the changelog if this function is renamed in the future).